configure.ac

```diff
@@ -131,8 +131,10 @@ AC_CHECK_FUNCS([bzero gettimeofday memmove memset mkdir strdup strrchr \
 LT_INIT
 AC_PROG_LIBTOOL
 
-# Decide if we're building on Windows early on.
 AM_CONDITIONAL([USING_WIN32], [test "x$host_os" = xmingw32])
+if test -z "${USING_WIN32_FALSE}"; then
```

Comment:
Apparently it should be that screwy: http://stackoverflow.com/questions/6679396/should-i-define-both-win32-and-win64-in-64bit-build